agent/rpm/pbench-agent.spec.j2

```diff
@@ -108,7 +108,6 @@ fi
 /%{installdir}/profile
 
 %ghost %attr(0400,pbench,pbench) /%{installdir}/id_rsa
-%ghost %attr(0664,pbench,pbench) /%{installdir}/config/pbench-agent.cfg
```

Comment:
Shouldn't we be doing the same thing as we do as `main`?  See https://github.com/distributed-system-analysis/pbench/blob/main/agent/rpm/pbench-agent.spec.j2#L131.

Reply:
Tracking in #2392.